pom.xml

```diff
@@ -78,6 +78,7 @@
         <module>embedded-mockserver</module>
         <module>embedded-solr</module>
         <module>embedded-cockroachdb</module>
+        <module>embedded-git</module>
```

Comment:
Please add embedded-git dependency to testcontainers-spring-boot-bom module.
https://github.com/PlaytikaOSS/testcontainers-spring-boot/blob/452fbae3cf5832addbd4104607aaa1ea92acd551/testcontainers-spring-boot-bom/pom.xml#L216

Reply:
done